### PR TITLE
 community: Add bio for Hulda and Eggert

### PR DESCRIPTION
**Pull request by eggertkarl**
Added Hulda and Eggert bios + pictures

### CODE REVIEW

File: website/docs/community/people.md [hunk 1]
```diff
@@ -4,4 +4,25 @@ title: People
...
+
+![Hulda Hvönn Kristinsdóttir](../../static/img/people/hulda.jpg)
+
+Hi!
+I'm Hulda. I have a Bachelors degree in Mathematics and Mathematical Education and a Masters degree in Upper Secondary Education of Mathematics. My role in the OpenEdu project is to review the content of the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland and adapt it to the new system developed by Open Education Hub.
+
+I work full time as a math teacher at the Commercial College of Iceland and as a part time teacher at the University of Iceland. I particularly enjoy teaching calculus, probability and statistics but I am also interested in the field of combinatorics.
+
+I spend most of my spare time taking care of my indoor jungle, doodling and weight lifting.
+
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
```

File: website/docs/community/people.md [hunk 2]
```diff
@@ -4,4 +4,25 @@ title: People
...
+
+## Eggert Karl Hafsteinsson
+
+![Eggert Karl Hafsteinsson](../../static/img/people/eggert.png)
+
+Hi!
+My name is Eggert Karl. I'm a math educator. I have a Masters degree in Mathematics (Multivariate Complex Analysis), Bachelors degrees in Applied Mathematics as well as Computer Science and have an additional diploma in Upper Secondary School Education. My work in the OpenEdu project consists mainly of adapting content developed for the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland for the new framework that we are developing. In addition, I also participate in development discussion, where I try to bring an educator's point of view, as well as how we can leverage past experiments, such as [tutor-web](https://tutor-web.net/).
+
+I teach various mathematics courses at the University of Iceland, in either main or assitant roles. These are usually calculus courses, though they can also included topics in complex analysis, partial differential equations, numerical analysis and more.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
I teach various mathematics courses at the University of Iceland, in either main or assistant roles. These are usually Calculus courses, though they can also included topics in Complex Analysis, Partial Differential Equations, Numerical Analysis and more.
```

**Review by lizababu — COMMENTED** [review]
Just small comments :D

File: website/docs/community/people.md [hunk 3]
```diff
@@ -4,4 +4,25 @@ title: People
...
 
-This is content
+## Hulda Hvönn Kristinsdóttir
+
+![Hulda Hvönn Kristinsdóttir](../../static/img/people/hulda.jpg)
+
+Hi!
+I'm Hulda. I have a Bachelors degree in Mathematics and Mathematical Education and a Masters degree in Upper Secondary Education of Mathematics. My role in the OpenEdu project is to review the content of the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland and adapt it to the new system developed by Open Education Hub.
+
+I work full time as a math teacher at the Commercial College of Iceland and as a part time teacher at the University of Iceland. I particularly enjoy teaching calculus, probability and statistics but I am also interested in the field of combinatorics.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
I work full time as a math teacher at the Commercial College of Iceland and as a part time teacher at the University of Iceland. I particularly enjoy teaching Calculus, Probability and Statistics, but I am also interested in the field of Combinatorics.
```

File: website/docs/community/people.md [hunk 4]
```diff
@@ -4,4 +4,24 @@ title: People
...
+Hi!
+I'm Hulda. I have a Bachelors degree in Mathematics and Mathematical Education and a Masters degree in Upper Secondary Education of Mathematics. My role in the OpenEdu project is to review the content of the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland and adapt it to the new system developed by Open Education Hub.
+
+I work full time as a math teacher at the Commercial College of Iceland and as a part time teacher at the University of Iceland. I particularly enjoy teaching Calculus, Probability and Statistics, but I am also interested in the field of Combinatorics.
+
+I spend most of my spare time taking care of my indoor jungle, doodling and weight lifting.
+
+## Eggert Karl Hafsteinsson
+
+![Eggert Karl Hafsteinsson](../../static/img/people/eggert.png)
```
Comment (razvand) [inline website/docs/community/people.md:22]: Serial killer pic ... DONE

File: website/docs/community/people.md [hunk 5]
```diff
@@ -4,4 +4,24 @@ title: People
 
 # Project Team
 
-This is content
+## Hulda Hvönn Kristinsdóttir
+
+![Hulda Hvönn Kristinsdóttir](../../static/img/people/hulda.jpg)
+
+Hi!
+I'm Hulda. I have a Bachelors degree in Mathematics and Mathematical Education and a Masters degree in Upper Secondary Education of Mathematics. My role in the OpenEdu project is to review the content of the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland and adapt it to the new system developed by Open Education Hub.
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
I'm Hulda.
I have a Bachelors degree in Mathematics and Mathematical Education and a Masters degree in Upper Secondary Education of Mathematics.
My role in the OpenEdu project is to review the content of the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland and adapt it to the new system developed by Open Education Hub.
```

File: website/docs/community/people.md [hunk 6]
```diff
@@ -4,4 +4,24 @@ title: People
...
+
+I work full time as a math teacher at the Commercial College of Iceland and as a part time teacher at the University of Iceland. I particularly enjoy teaching Calculus, Probability and Statistics, but I am also interested in the field of Combinatorics.
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
I work full time as a math teacher at the Commercial College of Iceland and as a part time teacher at the University of Iceland.
I particularly enjoy teaching Calculus, Probability and Statistics, but I am also interested in the field of Combinatorics.
```

File: website/docs/community/people.md [hunk 7]
```diff
@@ -4,4 +4,24 @@ title: People
...
+
+I spend most of my spare time taking care of my indoor jungle, doodling and weight lifting.
+
```
Comment (razvand) [inline website/docs/community/people.md]: Remove extra blank line.

File: website/docs/community/people.md [hunk 8]
```diff
@@ -4,4 +4,24 @@ title: People
...
+## Eggert Karl Hafsteinsson
+
+![Eggert Karl Hafsteinsson](../../static/img/people/eggert.png)
+
+Hi!
+My name is Eggert Karl. I'm a math educator. I have a Masters degree in Mathematics (Multivariate Complex Analysis), Bachelors degrees in Applied Mathematics as well as Computer Science and have an additional diploma in Upper Secondary School Education. My work in the OpenEdu project consists mainly of adapting content developed for the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland for the new framework that we are developing. In addition, I also participate in development discussion, where I try to bring an educator's point of view, as well as how we can leverage past experiments, such as [tutor-web](https://tutor-web.net/).
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
My name is Eggert Karl.
I'm a math educator.
I have a Masters degree in Mathematics (Multivariate Complex Analysis), Bachelors degrees in Applied Mathematics as well as Computer Science and have an additional diploma in Upper Secondary School Education.
My work in the OpenEdu project consists mainly of adapting content developed for the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland for the new framework that we are developing.
In addition, I also participate in development discussion, where I try to bring an educator's point of view, as well as how we can leverage past experiments, such as [tutor-web](https://tutor-web.net/).
```

File: website/docs/community/people.md [hunk 9]
```diff
@@ -4,4 +4,24 @@ title: People
...
+
+I teach various mathematics courses at the University of Iceland, in either main or assistant roles. These are usually Calculus courses, though they can also included topics in Complex Analysis, Partial Differential Equations, Numerical Analysis and more.
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
I teach various Mathematics courses at the University of Iceland, in either main or assistant roles.
These are usually Calculus courses, though they can also included topics in Complex Analysis, Partial Differential Equations, Numerical Analysis and more.
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @eggertkarl . I made some minor suggestions.

Please use one sentence on one line. This will make reviews and updates easier. Diffs are line-centric and it helps to have one sentence on one line.

Also, please create two commits for this PR: one commit adds info about yourself and another one about Hulda. It's good practice to have one commit do one thing. [This](https://www.atlassian.com/git/tutorials/rewriting-history) is a very nice tutorial on editing commits.

**Comment by eggertkarl** [comment]
@razvand So, I read the tutorial and was trying to rebase, but it kept opening up an editor and wasn't interactive, as I thought it was supposed to be. I ended up backing up my changes, hard resetting the HEAD back to when I forked (found with reflog), then redoing my changes in two passes, making sure to commit between passes so the first commit is only adding Hulda's bio+picture, then the second pass is adding my bio+picture. 

The only other commit is something that Liza did and I seem to have accepted (though, I don't actually remember doing that...). Is the final result good enough now, or are there more things I can do?

File: website/docs/community/people.md [hunk 10]
```diff
@@ -19,4 +19,34 @@ I love spending time with students, teaching and finding ways to improve the cur
...
+## Eggert Karl Hafsteinsson
+
+![Eggert Karl Hafsteinsson](../../static/img/people/eggert.png)
+
+Hi!
+My name is Eggert Karl.
+I'm a math educator.
+I have a Masters degree in Mathematics (Multivariate Complex Analysis), Bachelors degrees in Applied Mathematics as well as Computer Science and have an additional diploma in Upper Secondary School Education.
+My work in the OpenEdu project consists mainly of adapting content developed for the Computing and Calculus for Applied Statistics (CCAS) course at the University of Iceland for the new framework that we are developing.
+In addition, I also participate in development discussion, where I try to bring an educator's point of view, as well as how we can leverage past experiments, such as [tutor-web](https://tutor-web.net/).
```
Comment (RazvanN7) [inline website/docs/community/people.md:29]:
```suggestion
In addition, I also participate in development discussions, where I try to bring an educator's point of view, as well as how we can leverage past experiments, such as [tutor-web](https://tutor-web.net/).
```

File: website/docs/community/people.md [hunk 11]
```diff
@@ -19,4 +19,34 @@ I love spending time with students, teaching and finding ways to improve the cur
...
+
+I teach various Mathematics courses at the University of Iceland, in either main or assistant roles.
+These are usually Calculus courses, though they can also included topics in Complex Analysis, Partial Differential Equations, Numerical Analysis and more.
```
Comment (RazvanN7) [inline website/docs/community/people.md:32]:
```suggestion
These are usually Calculus courses, though they can also include topics in Complex Analysis, Partial Differential Equations, Numerical Analysis and more.
```

**Review by RazvanN7 — COMMENTED** [review]
Some minor typo corrections.

**Pull request closed without merging**